packages/hdwallet-portis/src/portis.ts

```diff
@@ -6,17 +6,6 @@ import type Web3 from "web3";
 import * as btc from "./bitcoin";
 import * as eth from "./ethereum";
 
-// We might not need this. Leaving it for now to debug further
-class PortisTransport extends core.Transport {
-  public async getDeviceID() {
-    return "portis:0";
```

Comment:
are you sure axiom does not use this?

Reply:
yep; it's not referenced anywhere in axiom or chain adapters